core/schemas/bifrost.go

```diff
@@ -683,6 +683,12 @@ type BifrostError struct {
 	StatusCode     *int          `json:"status_code,omitempty"`
 	Error          ErrorField    `json:"error"`
 	AllowFallbacks *bool         `json:"-"` // Optional: Controls fallback behavior (nil = true by default)
+	StreamError    *StreamError  `json:"-"`
```

Comment:
can we change the name to something like StreamConfig? because its already inside Error and it doesnt contain any error as such